media/wav.ksy

```diff
@@ -0,0 +1,272 @@
+meta:
+  id: wav
+  file-extension: wav
+  title: Microsoft WAVE audio file
+  endian: le
+  license: BSD-3-Clause-Attribution
```

Comment:
This license probably needs adding a license text to the file, i.e. that "Redistribution and use in source and binary forms, with or without modification, are permitted... EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE."?